ui/store/actions.js

```diff
@@ -2203,6 +2203,16 @@ export function setOpenSeaEnabled(val) {
   };
 }
 
+export function detectCollectibles() {
```

Comment:
This is such a common pattern, it seems like we ought to have a function for this... but I guess that's something for another PR.

Reply:
agreed! But yeah for another PR 😄